pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1626,11 +1595,6 @@ func (s *AWSCloud) createTags(request *ec2.CreateTagsInput) (*ec2.CreateTagsOutp
 func (s *AWSCloud) EnsureTCPLoadBalancer(name, region string, publicIP net.IP, ports []*api.ServicePort, hosts []string, affinity api.ServiceAffinity) (*api.LoadBalancerStatus, error) {
 	glog.V(2).Infof("EnsureTCPLoadBalancer(%v, %v, %v, %v, %v)", name, region, publicIP, ports, hosts)
```

Comment:
Rather than ignoring region, I'd be happier if we had a check like

```
if region != s.region {
   return nil, fmt.Errorf("region specified was unexpected: %s vs %s", region, s.region)
}
```